examples/placeholder/linux/include/TestCommand.h

```diff
@@ -123,13 +122,14 @@ class TestCommand : public TestRunner,
             chip::DeviceLayer::PlatformMgr().ScheduleWork(ScheduleNextTest, arg);
             break;
         }
+        chip::DeviceLayer::PlatformMgr().RemoveEventHandler(OnPlatformEvent, arg);
```

Comment:
This needs to be in the case where we get the event we are waiting for, not out here where other events will trigger it....

Reply:
I was think about this incorrectly. That makes sense! Iff we receive the event we are expecting only then remove the event handler.